database/factories/CourseFactory.php

```diff
@@ -19,7 +19,7 @@ public function definition(): array
         return [
             'name' => $this->faker->word(),
             'abbreviation' => $this->faker->lexify('????'),
-            'color' => $this->faker->word(),
+            'classes' => $this->faker->word(),
```

Comment:
`classes` should not be set to a random word, because they represent tailwind classes.
Instead a valid tailwind-class should be used (see examples in [`CourseSeeder.php`](https://github.com/fsr5-fhaachen/portals/blob/main/database/seeders/CourseSeeder.php)).